audited.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rubocop', '~> 0.54.0'
   gem.add_development_dependency 'rspec-rails', '~> 3.5'
   gem.add_development_dependency 'single_cov'
+  gem.add_development_dependency 'sprockets', '< 4.0'
```

Comment:
This fixes the build. I'm not familiar with how the test rails app works, so I just pinned this for now.